### PR TITLE
datatable dependabot warning fixed 

### PR DESCRIPTION
**Pull request by sagarsearce**
1. Packages / libraries updated to resolve the security warnings (Dependabot alerts)
 - datatable.net 
 - ws, 
 - tmpl
 -  path-parser
 -  ansi-regex
2.  Few code blocks updated for better code formatting and indentation

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+        href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css"
+        integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
     <link rel="icon" href="Icons/Icons/google-spanner-logo.png" type="image/icon type">
-    <link rel='stylesheet' href='fonts/glyphicons-halflings-regular.woff2'></link>
+    <link rel='stylesheet' href='fonts/glyphicons-halflings-regular.woff2'>
+    </link>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap|Material+Icons" rel="stylesheet">
+    <link
```
Comment (bharadwaj-aditya) [inline frontend/index.html:15]: can you check the formatting here ? It looks like whitespaces have been introduced

File: frontend/index.html [hunk 2]
```diff
@@ -42,12 +48,16 @@
...
 </body>
-    
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://unpkg.com/popper.js@1.12.6/dist/umd/popper.js" integrity="sha384-fA23ZRQ3G/J53mElWqVJEGJzU0sTs+SvzG8fXVWP+kJQ1lwFAOkcUOysnlKJC33U" crossorigin="anonymous"></script>
-    <script src="https://unpkg.com/bootstrap-material-design@4.1.1/dist/js/bootstrap-material-design.js" integrity="sha384-CauSuKpEqAFajSpkdjv3z9t8E7RlpJ1UP0lKM/+NdtSarroVKu069AlsRPKkFBz9" crossorigin="anonymous"></script>
-    <script type="text/javascript" src="third_party/jquery.dataTables.min.js"></script>
-    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
-    <script type="module" src="js/app.js"></script>
+
+<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> -->
```
Comment (bharadwaj-aditya) [inline frontend/index.html:52]: please remove if not needed

**Review by bharadwaj-aditya — CHANGES_REQUESTED** [review]
Some formatting changes.
Where have the files that are being added been taken from ? We need to check the licenses for that code.

**Comment by sagarsearce** [comment]
> data table files taken from https://datatables.net/download/

>  license link is https://datatables.net/license/ same information will be found in jquery.dataTables@1.11.3.min.js file.

File: frontend/index.html (hunk 2 above)
Comment (sagarsearce) [reply frontend/index.html:52]: will remove this line.

File: frontend/index.html (hunk 1 above)
Comment (sagarsearce) [reply frontend/index.html:15]: We are using Prettier here. It breaks the line if it has more than 80 (default) characters. this look good to me but we can update the config if you want to.

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

File: frontend/index.html (hunk 2 above)
Comment (bharadwaj-aditya) [reply frontend/index.html:52]: Please remove the commented code before merging

Comment (sagarsearce) [reply frontend/index.html]: removed.

**Pull request merged**